stan/math/prim/scal/fun/grad_reg_inc_gamma.hpp

```diff
@@ -31,7 +31,8 @@ namespace stan {
         s *= - z / k;
         delta = s / ((k + a) * (k + a));
         if (isinf(delta))
-          throw domain_error("gradRegIncGamma not converging");
+          stan::math::domain_error("grad_reg_inc_gamma",
+                                   "is not converging", "", "");
```

Comment:
If std::domain_error is in scope, it is required.

I'd rather call our function throw_domain_error or
raise_domain_error if that's what it does.

> On Aug 6, 2016, at 5:16 PM, Krzysztof Sakrejda <notifications@github.com> wrote:
> 
> In stan/math/prim/scal/fun/grad_reg_inc_gamma.hpp:
> 
> > @@ -31,7 +31,8 @@ namespace stan {
> >          s *= - z / k;
> >          delta = s / ((k + a) * (k + a));
> >          if (isinf(delta))
> > -          throw domain_error("gradRegIncGamma not converging");
> > +          stan::math::domain_error("grad_reg_inc_gamma",
> > +                                   "is not converging", "", "");
> 
> Is that stan::math:: necessary?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>